packages/react-components/react-components/src/Concepts/Icons/IconTable.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from '@storybook/addon-docs';
+import { Spinner } from '@fluentui/react-spinner';
+
+<Meta title="Concepts/Developer/Icons/IconTable" />
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/183605249-d50200a2-1c3d-4eea-a22a-bed465ba88d1.png)

Can we use a better title there? For example:

```suggestion
<Meta title="Concepts/Developer/Icons/Icons list" />
```

```suggestion
<Meta title="Concepts/Developer/Icons/Icons catalog" />
```

Reply:
+1 for "Icons catalog"